packages/babel-parser/src/types.d.ts

```diff
@@ -1345,7 +1345,6 @@ export interface TsPropertySignature extends TsNamedTypeElementBase {
   type: "TSPropertySignature";
   readonly?: true;
   typeAnnotation?: TsTypeAnnotation;
-  initializer?: Expression;
```

Comment:
I checked the parser and it should not be actually used.

Reply:
It doesn't matter, thank you for any contribution! I leave this review just to facilitate other reviewers. :)